### PR TITLE
Updates dependencies to 2.7.0

### LINKED ISSUE #10: Bring all dependencies up to date

### PR DESCRIPTION
**Pull request by alvincrespo**
Updates addons dependencies.

## Changes

- Updates `package.json` and `bower.json``
- Makes appropriate updates to project files based on `ember init` diffs
- Fixes deprecation with addon's `lib/commands/headstart.js` file's override of `init`

### Enhancements

- Updates ember dependencies to 2.7.0 support

## Checklist

- [x] Unit tests
- [x] Documentation

Fixes #10

### CODE REVIEW

**Comment by alvincrespo** [comment]
Next steps here is to extend the unit tests.

**Comment by alvincrespo** [comment]
@cball Wanted to check in and see what you think would be a good level of testing for this PR? I think we should definitely extend the tests, but not sure if it should be part of this PR. Here is what I currently have set up:

https://github.com/echobind/ember-headstart/pull/15/commits/f55b905350f93028f0e8502493bd676408f82dac

I'm currently basing most of the on how ember-cli-deploy does testing:

- https://github.com/ember-cli-deploy/ember-cli-deploy/blob/master/node-tests/unit/index-test.js
- https://github.com/ember-cli-deploy/ember-cli-deploy/blob/master/index.js
- https://github.com/ember-cli-deploy/ember-cli-deploy/issues/408

File: package.json [hunk 1]
```diff
@@ -9,44 +9,50 @@
   "scripts": {
     "build": "ember build",
     "start": "ember server",
-    "test": "ember try:testall"
+    "try": "ember try:each",
+    "test": "node node_modules/mocha/bin/mocha 'node-tests/**/*-test.js'"
   },
```
Comment (cball) [inline package.json]: Let's config similar to ember-writer: https://github.com/echobind/ember-writer/blob/master/package.json

File: node-tests/unit/index-test.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  describe('includedCommands', function() {
+    it('returns a headstart command', function() {
+      assert(
+        subject.includedCommands().headstart,
+        'returns an object with headstart key'
+      );
+    });
+  });
+});
```
Comment (cball) [inline node-tests/unit/index-test.js]: :tada:

File: .travis.yml [hunk 3]
```diff
@@ -21,14 +21,17 @@ matrix:
...
 install:
-  - npm install -g bower
   - npm install
   - bower install
 
 script:
-  - ember try $EMBER_TRY_SCENARIO test
+  # Usually, it's ok to finish the test scenario without reverting
+  #  to the addon's original dependency state, skipping "cleanup".
+  - ember try:one $EMBER_TRY_SCENARIO test --skip-cleanup
```
Comment (cball) [inline .travis.yml]: Would be good to move this to circle. Doesn't have to be part of this PR though

**Comment by alvincrespo** [comment]
@cball Ready for another round of review!

**Pull request merged**